content/docs/yaml-reference/story-runner-components/index.md

```diff
@@ -0,0 +1,57 @@
+---
+title: "YAML Runner Components"
+description: "Commmon Keys Used for the Yaml Runner"
+lead: "Common Keys Used for the Yaml Runner"
+date: 2021-06-09T11:39:03+01:00
+lastmod: 2021-06-09T11:39:03+01:00
+draft: false
+images: []
+menu: 
+  docs:
+    parent: "yaml-reference"
+weight: 71
+toc: true
+---
+
+{{< note >}}
+**Note:** To use the YAML Runner you will need to install the `jitsuin-archivist` python package.
+
+[Click here](https://python.rkvst.com/runner/index.html) for installation instructions.
+{{< /note >}}
+
+| **Key**            | **Value**                                                                                                                                                                                                                                                                                                       |
+|--------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| **action**         | Required for every operation, the action specifies what function will be performed.                                                                                                                                                                                                                             |
+| **description**    | Optional string that describes what the step is doing. For example, "Create the Asset My First Container".                                                                                                                                                                                                      |
+| **asset_label**    | For a series of steps run as one file, the asset label could be a friendly name used by later steps to refer back to an asset created in a previous step. If the asset already exists, this field may be used to reference the Asset ID in the form assets/add30235-1424-4fda-840a-d5ef82c4c96fa.               |
```

Comment:
Can we make it more clear that the assets/_asset ID_ is sourced from the existing asset? 

"assets/[_your_asset_ID_]" or similar, with the example?

Reply:
Yes, I updated all the example IDs throughout the YAML Reference to match the examples from our other sections - "assets/<asset-id>".